test/Prototypes/BigInt.swift

```diff
@@ -1450,7 +1456,7 @@ func testBinaryInit<T: BinaryInteger>(_ x: T) -> BigInt {
 }
 
 func randomBitLength() -> Int {
-  return Int(arc4random_uniform(1000) + 2)
+  return Int.random(in: 2...1000)
```

Comment:
I don't think it matters much in this particular case, but the random value range is different. The old one includes 1001, the new one does not.